Main.java

```diff
@@ -1,7 +1,7 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class main {
```

Comment:
could you also move the main file into a folder called "ui"? intellij isn't recognizing it as a source file, so yeah lool